tools/bin/cassandra.in.sh

```diff
@@ -31,7 +31,7 @@ CLASSPATH="$CASSANDRA_CONF"
 # it's just used here in constructing the classpath.
 if [ -d $CASSANDRA_HOME/build ] ; then
     #cassandra_bin="$CASSANDRA_HOME/build/classes/main"
-    cassandra_bin=`ls -1 $CASSANDRA_HOME/build/apache-cassandra*.jar`
+    cassandra_bin=`ls -1 $CASSANDRA_HOME/build/apache-cassandra*.jar | paste -sd ":" -`
```

Comment:
what is this `paste`? @dmi0

Reply:
`ls -1 $CASSANDRA_HOME/build/apache-cassandra*.jar` produces a list of files separated by "\n", which results in the `cassandra_bin` variable omitting some jar files. This issue caused the unit test to fail.

[ `paste -sd ":"`](https://linuxcommandlibrary.com/man/paste) joins all the file names into a single line, using ":" as a delimiter.

@smiklosovic

Reply:
wow ... could you please put here example of `cassandra_bin` with and without this change?

Reply:
All I see is this:

    ls -1 build/apache-cassandra*.jar | paste -sd ":" -
    build/apache-cassandra-5.0-beta1-SNAPSHOT.jar:build/apache-cassandra-5.0-beta1-SNAPSHOT-sources.jar

I do not think we need source on the class path? What additional jars do you see?

Reply:
Sorry, my bad. Just double checked, original one works for me as well (was confused by different jar versions)
Reverting this.